AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/linked-participant.model.ts

```diff
@@ -0,0 +1,12 @@
+export class LinkedParticipantModel {
+    id?: string | undefined;
+    participantId?: string | undefined;
+    participantEmail?: string | undefined;
+    linkedParticipantId?: string | undefined;
+    linkedParticipantEmail?: string | undefined;
+    linkType?: LinkedParticipantType;
+}
+
+export enum LinkedParticipantType {
```

Comment:
Is this a duplication of the enum above?

Reply:
removed this enum class